experiments/AtmosLES/dycoms.jl

```diff
@@ -306,7 +306,7 @@ function config_dycoms(FT, N, resolution, xmax, ymax, zmax)
     α_z           = FT(1)
     z_i           = FT(840)
     ρ_i           = FT(1.13)
-    D_subsidence  = FT(0) # 0 for stable testing, 3.75e-6 in practice
+    D_subsidence  = FT(3.75e-6) # 0 for stable testing, 3.75e-6 in practice
```

Comment:
The intent here is to modify this to be consistent with your stable `offline` simulations at `timeend = 14400s` , which means this comment about stable testing vs practice can be removed.

Reply:
```suggestion
    D_subsidence  = FT(3.75e-6) 
```